packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -69,6 +67,9 @@ class ConfigManager implements IConfigManager {
       if (state.loadOptions.value.configUrl) {
         state.lifecycle.sourceConfigUrl.value = `${state.loadOptions.value.configUrl}/sourceConfig/?p=${MODULE_TYPE}&v=${APP_VERSION}&writeKey=${state.lifecycle.writeKey.value}&lockIntegrationsVersion=${lockIntegrationsVersion}`;
       }
+      // set cookie consent option provided as load option in consent state
+      state.consents.cookieConsentOptions.value =
```

Comment:
Let's use the same name as the options field.

Reply:
Now we are only storing the name of the consent manager plugin from option.